src/create_disk.sh

```diff
@@ -426,6 +426,9 @@ echo "set ignition_network_kcmdline='${firstboot_kargs}'" > $rootfs/boot/ignitio
 chattr +i $rootfs
 
 fstrim -a -v
+# Ensure the filesystem journal is flushed
+mount -o remount,ro $rootfs
+xfs_freeze -f $rootfs
```

Comment:
Should this be conditionaled upon  `$rootfs_type`?
Apparently the rest of the code supports `xfs`, `ext4verity` and `luks`. I don't know what's `xfs_freeze` behavior in those cases.

Reply:
Yeah, some legacy here; the tool is still called `xfs_freeze` but from `man xfs_freeze`:

```
NOTES
...
       In Linux kernel version 2.6.29, the interface which XFS uses to freeze and unfreeze was elevated to the VFS, so that this tool can now be used on many other Linux filesystems.
```

And it's supported by ext4 for sure (luks is block level, so it's orthogonal to this).

Similarly, xfstests is now a generic filesystem testing tool and used by most other filesystems too.